oci/oci.go

```diff
@@ -95,6 +102,12 @@ type Runtime struct {
 	ctrStopTimeout           int64
 }
 
+// RuntimeHandler represents each item of the "crio.runtime.runtimes" TOML
```

Comment:
I'd rather use `crio.runtimes` directly in the config

Reply:
So we would add a RuntimesConfig everywhere? I'm asking because CRI-O config seems pretty well organized with sections `runtime`/`image`/`api`, and it feels that adding `crio.runtimes` would add a new category, which is related to `runtime`.

Reply:
I believe `runtime` is too generic to be a top level at this point cause now that we have `runtimes`, the other options should very well be `runtime_options`. But even in that case, it's not that good. I don't know :smile:

Reply:
nevermind this comment anymore